src/libsyntax/ext/tt/macro_rules.rs

```diff
@@ -276,7 +277,7 @@ pub fn compile(
                     if body.legacy { token::Semi } else { token::Comma },
                     def.span,
                 )),
-                op: quoted::KleeneOp::OneOrMore,
+                kleene: quoted::KleeneToken::new(quoted::KleeneOp::OneOrMore, DUMMY_SP),
```

Comment:
I don’t think this is correct. It should be a dummy span. There is no reasonable case where this span should be used, I think.

Reply:
There is actually [this comment](https://github.com/rust-lang/rust/blob/9a90d03ad171856dc016c2dcc19292ec49a8a26f/src/libsyntax/ext/tt/macro_rules.rs#L265) that says that those spans don't matter. So since `def.span` is used everywhere (but for the sequences), for coherence, I guess it makes sense to use `def.span` too.